README.rst

```diff
@@ -192,7 +192,7 @@ The following options can be configured on the server:
     **Discovery**
     discovery.client.refresh_interval          10m0s                                                                                                                                                                                                                                                                                                                                                                                                  Interval at which the client synchronizes with the Discovery Server; refreshing Verifiable Presentations of local DIDs and loading changes, updating the local copy. It only will actually refresh registrations of local DIDs that about to expire (less than 1/4th of their lifetime left). Specified as Golang duration (e.g. 1m, 1h30m).
     discovery.definitions.directory                                                                                                                                                                                                                                                                                                                                                                                                                   Directory to load Discovery Service Definitions from. If not set, the discovery service will be disabled. If the directory contains JSON files that can't be parsed as service definition, the node will fail to start.
-    discovery.server.definition_ids            []                                                                                                                                                                                                                                                                                                                                                                                                     IDs of the Discovery Service Definitions for which to act as server. If an ID does not map to a loaded service definition, the node will fail to start.
+    discovery.server.definitions               []                                                                                                                                                                                                                                                                                                                                                                                                     IDs of the Discovery Service Definitions for which to act as server. If an ID does not map to a loaded service definition, the node will fail to start.
```

Comment:
`definitions.directory` vs `server.definitions` looks a bit weird. should it be `definitions.server`?

Reply:
I changed `discovery.server.definitions to `ids` and `discovery.definitions.directory` to simply `discovery.definitions`